<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>ProposalDialog</class>
  <widget class="QDialog" name="ProposalDialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>126</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Budget Proposal</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
        <number>11</number>
      </property>
      <property name="topMargin">
        <number>11</number>
      </property>
      <property name="rightMargin">
        <number>11</number>
      </property>
      <property name="bottomMargin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>11</number>
      </property>
      <item>
        <widget class="QLabel" name="infoLabel">
          <property name="text">
            <string>Please fill out the form below to create a budget proposal.  Before beginning make sure your wallet is unlocked to pay the budget proposal fee of 5 BWK.  After the proposal is submitted you will be shown the transaction hash field and will need to wait for confirmations on the transaction before being able to submit the proposal for voting.  The screen will update to help guide you, it is never recommended to close the dialog, it will close once completed automatically.</string>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="leftMargin">
            <number>11</number>
          </property>
          <property name="topMargin">
            <number>11</number>
          </property>
          <property name="rightMargin">
            <number>11</number>
          </property>
          <property name="bottomMargin">
            <number>11</number>
          </property>
          <property name="spacing">
            <number>11</number>
          </property>
          <item row="0" column="0">
            <widget class="QLabel" name="label">
              <property name="text">
                <string>&amp;Proposal Name:</string>
              </property>
              <property name="buddy">
                <cstring>nameEdit</cstring>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QValidatedLineEdit" name="nameEdit">
            </widget>
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="label_2">
              <property name="text">
                <string>&amp;Proposal URL:</string>
              </property>
              <property name="buddy">
                <cstring>urlEdit</cstring>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="QValidatedLineEdit" name="urlEdit">
            </widget>
          </item>
          <item row="2" column="0">
            <widget class="QLabel" name="label_3">
              <property name="text">
                <string>&amp;Payments (Months):</string>
              </property>
              <property name="buddy">
                <cstring>paymentsEdit</cstring>
              </property>
            </widget>
          </item>
          <item row="2" column="1">
            <widget class="QValidatedLineEdit" name="paymentsEdit">
            </widget>
          </item>
          <item row="3" column="0">
            <widget class="QLabel" name="label_7">
              <property name="text">
                <string>&amp;Super Block:</string>
              </property>
              <property name="buddy">
                <cstring>blockEdit</cstring>
              </property>
            </widget>
          </item>
          <item row="3" column="1">
            <widget class="QValidatedLineEdit" name="blockEdit">
            </widget>
          </item>
          <item row="4" column="0">
            <widget class="QLabel" name="label_4">
              <property name="text">
                <string>&amp;Payment Address:</string>
              </property>
              <property name="buddy">
                <cstring>addressEdit</cstring>
              </property>
            </widget>
          </item>
          <item row="4" column="1">
            <widget class="QValidatedLineEdit" name="addressEdit">
            </widget>
          </item>
          <item row="5" column="0">
            <widget class="QLabel" name="label_5">
              <property name="text">
                <string>&amp;Payment Amount (Month):</string>
              </property>
              <property name="buddy">
                <cstring>amountEdit</cstring>
              </property>
            </widget>
          </item>
          <item row="5" column="1">
            <widget class="QValidatedLineEdit" name="amountEdit">
            </widget>
          </item>
          <item row="6" column="0">
            <widget class="QLabel" name="hashLabel">
              <property name="text">
                <string>&amp;Proposal TXID:</string>
              </property>
              <property name="buddy">
                <cstring>hashEdit</cstring>
              </property>
            </widget>
          </item>
          <item row="6" column="1">
            <widget class="QValidatedLineEdit" name="hashEdit">
            </widget>
          </item>
          <item row="7" column="1">
            <widget class="QLabel" name="confirmLabel">
              <property name="text">
                <string></string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>  
        <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
            <widget class="QtMaterialFlatButton" name="acceptButton" native="true">
              <property name="text" stdset="0">
                <string>Submit Proposal</string>
              </property>
              <property name="autoDefault" stdset="0">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QtMaterialFlatButton" name="cancelButton" native="true">
              <property name="text" stdset="0">
                <string>Cancel</string>
              </property>
              <property name="autoDefault" stdset="0">
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <resources>
    <include location="../bulwark.qrc"/>
  </resources>
  <connections>
    <connection>
      <sender>buttonBox</sender>
      <signal>accepted()</signal>
      <receiver>ProposalDialog</receiver>
      <slot>accept()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>20</x>
          <y>20</y>
        </hint>
        <hint type="destinationlabel">
          <x>20</x>
          <y>20</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>buttonBox</sender>
      <signal>rejected()</signal>
      <receiver>ProposalDialog</receiver>
      <slot>reject()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>20</x>
          <y>20</y>
        </hint>
        <hint type="destinationlabel">
          <x>20</x>
          <y>20</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>
